src/tests/ftest/rebuild/cascading_failures.py

```diff
@@ -110,7 +111,7 @@ def test_simultaneous_failures(self):
         self.mode = "simultaneous"
         self.execute_rebuild_test()
 
-    @skipForTicket("DAOS-2469")
+    @skipForTicket("DAOS-6256")
```

Comment:
DAOS-6256 is shows resolved.
If you are skipping this test, either re-open DAOS-6256 is the issue was not resolved or open a ticket if the test is failing for a different issue.

Reply:
DAOS-6256 still reproduceable, have reopened the JIRA.